pkg/scorecard_test.go

```diff
@@ -187,3 +191,128 @@ func TestRunScorecard(t *testing.T) {
 		})
 	}
 }
+
+func TestExperimentalRunProbes(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		uri       string
+		commitSHA string
+		probes    []string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    ScorecardResult
+		wantErr bool
+	}{
+		{
+			name: "empty commits repos should return repo details but no checks",
+			args: args{
+				uri:       "github.com/ossf/scorecard",
+				commitSHA: "1a17bb812fb2ac23e9d09e86e122f8b67563aed7",
+				probes:    []string{"fuzzedWithOSSFuzz"},
+			},
+			want: ScorecardResult{
+				Repo: RepoInfo{
+					Name:      "github.com/ossf/scorecard",
+					CommitSHA: "1a17bb812fb2ac23e9d09e86e122f8b67563aed7",
+				},
+				RawResults: checker.RawResults{
+					Metadata: checker.MetadataData{
+						Metadata: map[string]string{
+							"repository.defaultBranch": "main",
+							"repository.host":          "github.com",
+							"repository.name":          "ossf/scorecard",
+							"repository.sha1":          "1a17bb812fb2ac23e9d09e86e122f8b67563aed7",
+							"repository.uri":           "github.com/ossf/scorecard",
+						},
+					},
+				},
+				Scorecard: ScorecardInfo{
+					CommitSHA: "unknown",
+				},
+				Findings: []finding.Finding{
+					{
+						Probe:   "fuzzedWithOSSFuzz",
+						Message: "no OSSFuzz integration found",
+						Remediation: &probe.Remediation{
+							Text: fmt.Sprintf("%s%s%s\n%s%s",
+								"Follow the steps in ",
+								"https://github.com/google/oss-fuzz ",
+								"to integrate fuzzing for your project.",
+								"Over time, try to add fuzzing for more ",
+								"functionalities of your project."),
```

Comment:
is this just to avoid the `lll` linter? We should really turn that off for tests. Might be cleaner to use back ticks:
```
    foo := `some really
long long
line`
```

Reply:
> is this just to avoid the lll linter?

Yes

Reply:
Changed to ignoring the `Text` and `Markdown` fields of the `Remediation` here: https://github.com/ossf/scorecard/pull/3688/commits/ef1ab210ebc413b8db7910653c5ea62dc162267a